wikiwho_api/settings_wmcloud.py

```diff
@@ -18,12 +18,13 @@
 
 ONLY_READ_ALLOWED = False
 
-ACTIONS_LANGUAGES = ['tr', 'eu', 'es', 'de', 'en']
+ACTIONS_LANGUAGES = ['tr', 'eu', 'es', 'de', 'en', 'fr']
 PICKLE_FOLDER_EN = '/pickles/en'
 PICKLE_FOLDER_EU = '/pickles/eu'
 PICKLE_FOLDER_ES = '/pickles/es'
 PICKLE_FOLDER_DE = '/pickles/de'
 PICKLE_FOLDER_TR = '/pickles/tr'
+PICKLE_FOLDER_FR = '/pickles-02/fr'
```

Comment:
Some comments in this file about the storage strategy might be helpful (ie, dividing this list of folders explicitly according to which volume they are on.

Reply:
Good idea, done!